tests/supervisors/test_multiprocess.py

```diff
@@ -34,6 +34,7 @@ def new_function():
                 "-c",
                 f"from {module} import {name}; {name}.__wrapped__()",
             ],
+            timeout=10,
```

Comment:
Why do we need the timeout?

Reply:
Oh, this was added earlier when I was investigating why it was hanging. No need for it now, I'll remove it later